.gitignore

```diff
@@ -47,3 +47,17 @@ vmlinux
 # Python Virtual Env
 *.env
 *.venv
+
+### Vim ###
+# swap
```

Comment:
This is fine but you might want to set vim to save swap in `/tmp` or something instead. That way you don't have to worry about this kind of thing in every repo.